packages/scaffold/src/partials/w3m-all-wallets-list/index.ts

```diff
@@ -145,7 +145,11 @@ export class W3mAllWalletsList extends LitElement {
 
   private onConnectWallet(wallet: WcWallet) {
     const { connectors } = ConnectorController.state
-    const connector = connectors.find(({ explorerId }) => explorerId === wallet.id)
+
+    const connector = connectors.find(
+      ({ explorerId, info, id }) =>
+        explorerId === wallet.id || info?.rdns === wallet.rdns || wallet.rdns === id
+    )
```

Comment:
Can we add a comment here explaining whats happening

Reply:
I refactored the code to make it simpler but added a comment. Basically either matches by explorer ID or RDNS 🖖